### PR TITLE
Support Wear screenshots

### PR DESCRIPTION
**Pull request by yschimke**
Not sure about the right API, particularly to support both full screen screenshots, and component screenshots without a crop.

image

### CODE REVIEW

**Comment by takahirom** [comment]
👀

**Comment by yschimke** [comment]
I'm struggling to get a transparent background, so still working on it.

**Comment by yschimke** [comment]
Examples of current RNG screenshots. https://github.com/joreilly/Confetti/blob/main/wearApp/snapshot/SessionsScreenTest/sessionsScreen_Colors_MobileAndroid__reference.png

**Comment by takahirom** [comment]
Thanks, I think this issue might be the cause. 👀 
https://github.com/takahirom/roborazzi/issues/34

File: roborazzi/src/main/java/com/github/takahirom/roborazzi/capture.kt [hunk 1]
```diff
@@ -285,7 +285,8 @@ data class RoborazziOptions(
   }
 
   data class RecordOptions(
-    val resizeScale: Double = 1.0
+    val resizeScale: Double = 1.0,
+    val fullScreen: Boolean = true
```
Comment (takahirom) [inline roborazzi/src/main/java/com/github/takahirom/roborazzi/capture.kt:289]: I believe adding the parameter to RecordOptions is the right choice! However, the name "fullscreen" might be a bit confusing. Users could potentially think it refers to a navigation bar or status bar. I think the name "applyDeviceCrop: Boolean = true" is a better alternative and should be clearer for users.
While "fullscreen" is useful for other features and might be a good name, do you have any suggestions for features that would work well with fullscreen?

Comment (yschimke) [reply roborazzi/src/main/java/com/github/takahirom/roborazzi/capture.kt]: Done.  I think eventually it should be true for full screen tests, and defaulted to true.  But for now, changing to false until it works end to end.

File: app/src/test/java/com/github/takahirom/roborazzi/sample/ComposeTest.kt [hunk 2]
```diff
@@ -33,4 +54,26 @@ class ComposeTest {
         .performClick()
     }
   }
+
+  @Test
+  @Config(
+    sdk = [30],
+    qualifiers = "w221dp-h221dp-small-notlong-round-watch-xhdpi-keyshidden-nonav"
```
Comment (takahirom) [inline app/src/test/java/com/github/takahirom/roborazzi/sample/ComposeTest.kt:61]: 👍

**Review by takahirom — APPROVED** [review]
I think it's better to include this implementation in the library. This pull request is marked as "For discussion." However, can I merge it?

File: app/src/test/java/com/github/takahirom/roborazzi/sample/ComposeTest.kt (hunk 2 above)
Comment (yschimke) [reply app/src/test/java/com/github/takahirom/roborazzi/sample/ComposeTest.kt:61]: We may want to provide these as "const val" for the same devices that are provided in Android Studio previews.

**Pull request merged**